corehq/form_processor/submission_post.py

```diff
@@ -248,6 +249,7 @@ def run(self):
                     submission_type = 'device_log'
                     try:
                         process_device_log(self.domain, instance)
+                        send_device_logs_to_sumologic.delay(self.domain, instance)
```

Comment:
This won't get called for ICDS at present because of above `ignore_device_logs` check. For ICDS we want to call this but not the normal `process_device_log`.

Reply:
Oops, missed that. Thanks. https://github.com/dimagi/commcare-hq/pull/19597/commits/ba216a1eda47464f90d35aed28ff63b601c44874